examples/bri-3/src/bri/workgroup/workgroups/capabilities/createWorkgroup/createWorkgroupCommand.handler.ts

```diff
@@ -0,0 +1,42 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { WorkgroupAgent } from '../../agents/workgroups.agent';
+import { WorkgroupStorageAgent } from '../../agents/workgroupStorage.agent';
+import { CreateWorkgroupCommand } from './createWorkgroup.command';
+
+@CommandHandler(CreateWorkgroupCommand)
+export class CreateWorkgroupCommandHandler
+  implements ICommandHandler<CreateWorkgroupCommand>
+{
+  constructor(
+    private agent: WorkgroupAgent,
+    private storageAgent: WorkgroupStorageAgent,
+  ) {}
+
+  async execute(command: CreateWorkgroupCommand) {
+    const administratorsToConnect =
```

Comment:
I would avoid leakage of specific ORM library language on this level. We could for with workgroupAdministrators and workgroupParticipants instead, on something similar that avoids the word _connect_